app/server/lib/ConfigBackendAPI.ts

```diff
@@ -0,0 +1,67 @@
+import * as express from 'express';
+import {expressWrap} from 'app/server/lib/expressWrap';
+
+import {IGristCoreConfig} from 'app/server/lib/configCore';
+import {getGlobalConfig} from 'app/server/lib/globalConfig';
+
+import log from "app/server/lib/log";
+
+export class ConfigBackendAPI {
+  private static _doRestart = false;
+  private _config: IGristCoreConfig;
+
+  constructor() {
+      this._config = getGlobalConfig();
```

Comment:
Why do you need to run this function here? Can't it be lazy evaluated when it is actually needed?

Reply:
I reasoned that this function should be eagerly loaded at startup as getting the configuration object should happen once and stay around.

What change do you propose? To load the configuration object every time we handle an API request?

Reply:
Yes, exactly that. I see in the implementation, that it is already being cached there. So there is no point in adding a second layer of caching. I think we should avoid doing work in constructors, at all cost, only the necessary setup, and here, this constructor, actually reads some files from disk, which can be very surprising and cause bugs later.

Reply:
Okay, that makes sense, done.